versions.yaml

```diff
@@ -31,27 +47,3 @@ versions:
       pushgateway:
         version: '0.9.1'
         sha256: '6741f8e8f4d728a907991c7a9f0a7e32e1e9cf339a9795724412e54466f2e8e1'
-  - dir: '0/debian9/0.5'
```

Comment:
Those tags (0.5 and 0.4) are still available on GCP Marketplace. They must to be mark as depreciated.